### DISCUSSION TITLE
Stuttering mid-game

### DISCUSSION THREAD

**Discussion by Frous-s** [ppy/osu#16466] [Q&A]
Hi, so basically I had a short session on lazer(around 15 minutes), and on a few maps I had random stuttering mid game which made me throw my combo entirely. I'm entirely up to date with the osu version (2022.108) and my frame limiter is on unlimited, this issue was apparent on earlier versions but it seems to be more common, to about 1 per map play.

**Comment by Frous-s** [comment]
[network.log](https://github.com/ppy/osu/files/7875157/network.log)
[performance.log](https://github.com/ppy/osu/files/7875158/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7875159/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7875160/updater.log)
[database.log](https://github.com/ppy/osu/files/7875161/database.log)

logs of course

**Comment by peppy** [comment]
Using unlimited for anything but benchmarking is not recommended. Please use a sane frame limiter.

Also please post screenshots or video showing the stutters with frame graphs visible (Ctrl+F11 twice).
